apps/web/src/components/Logo/ChainLogo.tsx

```diff
@@ -1,33 +1,18 @@
-import { ChainId } from '@uniswap/sdk-core'
+import { ChainId } from '@hemilabs/sdk-core'
```

Comment:
This shouldn't be necessary, as we're fixing this on build time on [tsconfig](https://github.com/hemilabs/interface/blob/b48f3a38b18ceb7e6876fcdac085a00be39420ad/apps/web/tsconfig.json#L32) and [craco.config.cjs](https://github.com/hemilabs/interface/blob/b48f3a38b18ceb7e6876fcdac085a00be39420ad/apps/web/craco.config.cjs#L94). 
Do you get any errors?

Reply:
Right, it's not needed. Changed here: https://github.com/hemilabs/interface/pull/3/commits/86e5c778ba600c823948676982198f575c69d51b